homeassistant/components/sensor/netatmo.py

```diff
@@ -11,19 +11,31 @@
 from homeassistant.util import Throttle
 from homeassistant.loader import get_component
 
+# Fix for pylint too many statements error
+# pylint: disable=too-many-statements
```

Comment:
You shouldn't add this here at the top of the module, but local where the problem is. Put it on the next row directly after the docstring in the update method, near row 109. The disable statement has scope, so it's best to disable as little as possible, and it also makes it easier to see why it was needed.

The update method could be refactored to avoid the disable. It might be out of scope for this PR, but I would do a nested dict lookup and have a function that checks if data value is above inner dict value.